pkg/virt-api/rest/subresource_test.go

```diff
@@ -289,6 +313,114 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 			close(done)
 		})
 
+		It("should fail to ForceRestart if VirtualMachine is not in running state", func(done Done) {
+			request.PathParameters()["name"] = "testvm"
+			request.PathParameters()["namespace"] = "default"
+
+			request.Request.URL = &url.URL{}
+			q := request.Request.URL.Query()
+			q.Add("force", "True")
+			q.Add("graceperiod", "0")
+			request.Request.URL.RawQuery = q.Encode()
+
+			vm := v1.VirtualMachine{
+				Spec: v1.VirtualMachineSpec{
+					Running: &notRunning,
+				},
+			}
+
+			server.AppendHandlers(
+				ghttp.CombineHandlers(
+					ghttp.VerifyRequest("GET", "/apis/kubevirt.io/v1alpha3/namespaces/default/virtualmachines/testvm"),
+					ghttp.RespondWithJSONEncoded(http.StatusOK, vm),
+				),
+			)
+
+			app.RestartVMRequestHandler(request, response)
+
+			Expect(response.Error()).To(HaveOccurred())
+			Expect(response.StatusCode()).To(Equal(http.StatusForbidden))
+			// check the msg string that would be presented to virtctl output
+			Expect(response.Error().Error()).To(Equal("Halted does not support manual restart requests"))
+			close(done)
+		})
+
+		It("should Force restart VirtualMachine", func(done Done) {
```

Comment:
titles of "It" stanzas are inconsistent.
"Force restart" -> "ForceRestart"

Reply:
Corrected